starboard/BUILD.gn

```diff
@@ -51,6 +51,7 @@ group("gn_all") {
   if (sb_filter_based_player) {
     deps += [
       "//starboard/shared/starboard/player/filter/testing:player_filter_tests($starboard_toolchain)",
+      "//starboard/shared/starboard/player/filter/testing:player_filter_tests_install($starboard_toolchain)",
```

Comment:
A little confused by this. Why do you have to add the install target for player_filter_tests here? And if you really do should it replace the line above instead?

Reply:
It's a quirk of GN—I believe if nothing depends on a target built with a toolchain other than the default, it will not be in the ninja database.

I could probably just replace player_filter_tests above, but let's do that in a follow up as I have to make sure it isn't affected by b/296884941